_data/navigation.yml

```diff
@@ -340,3 +340,73 @@ es:
       twitter_impersonation:
         title: "Twitter impersonation"  # XXX
         url: "/en/twitter-impersonation"  # XXX
+tr:
+  home:
+    title: "Bitcoin Core"
+    url:   "/tr/"
+  about:
+    title: "Hakkında"
+    url:   "/tr/about/"
+  download:
+    title: "İndir"
+    url: "/tr/download/"
+  blog:
+    title: "blog"
+    url:   "/en/blog/"
+  releases:
+    title: "Versiyonlar"
+    url:   "/tr/releases/"
+  faq:
```

Comment:
You might have noticed, you can postpone FAQ for now,
there is a PR to remove them.

Reply:
I'm changing right now

Reply:
This is a dialogue created for a one line length code section
that only includes line 359, which reads: "faq". 😝

If GitHub or your browser or your mail viewer broke visuals,
that's on them. 😂

Reply:
FAQ I'm postponing for now. If there is a fix, change or improvement you want, feel free to point it out. 👍